### PR TITLE
Sno page cleanup

### PR DESCRIPTION
**Pull request by mbouzi**
mocks - https://www.figma.com/file/to8O98LoCFBm6hvAZmzDxo/Storj.io-Vanguard-Copy?node-id=1099%3A0
- changes at "storage-node-operator"

### CODE REVIEW

File: layouts/partials/storage-node-operator/main.html [hunk 1]
```diff
@@ -17,7 +17,8 @@ <h1 class="start-sharing-header">Start Sharing. Start Earning.</h1>
       </div>
     {{end}}
     </div>
-    <div class="spacer50"></div>
+    <div class="spacer40"></div>
```
Comment (VinozzZ) [inline layouts/partials/storage-node-operator/main.html]: Maybe we can just create a `spacer80` ?

File: layouts/partials/storage-node-operator/nodePerformance.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<div class="np-bg">
+  <div class="container">
+    <div class="row">
+      {{ with .Site.GetPage "nodePerformance"}}
+      <div class="col-md-4 col-xs-12">
+        <div class="spacer100"></div>
```
Comment (VinozzZ) [inline layouts/partials/storage-node-operator/nodePerformance.html]: same as above --> `spacer200`

**Review by VinozzZ — APPROVED** [review]

File: layouts/partials/storage-node-operator/main.html (hunk 1 above)
Comment (mbouzi) [reply layouts/partials/storage-node-operator/main.html]: added!

File: layouts/partials/storage-node-operator/nodePerformance.html (hunk 2 above)
Comment (mbouzi) [reply layouts/partials/storage-node-operator/nodePerformance.html]: done!

**Pull request merged**